test/tests/03-enterprise-licensing-tests/run.sh

```diff
@@ -0,0 +1,51 @@
+set pipefail
+
+if [[ "$EDITION" != "enterprise" ]]; then
+  exit 0
+fi
+
+msg_test "Check admin API is alive"
+assert_response "$KONG_ADMIN_URI" "200"
+
+msg_test "Create a service"
+assert_response "-d name=testservice -d url=http://mockbin $KONG_ADMIN_URI/services" "201"
+
+msg_test  "List services"
+assert_response "$KONG_ADMIN_URI/services" "200"
+
+msg_test "Create a route"
+assert_response "-d name=testroute -d paths=/ $KONG_ADMIN_URI/services/testservice/routes" "201"
+
+msg_test "List services"
```

Comment:
shall we test the "List routes" here

Reply:
added list routes in 4b4c703a51a4bcf64bcd8ced3173d44533b5c8a5